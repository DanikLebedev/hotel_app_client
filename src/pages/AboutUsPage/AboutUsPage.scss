.videoTag {
  position: fixed;
  right: 0;
  bottom: 0;
  min-width: 100%;
  min-height: 100%;
  z-index: -3;
  filter: blur(3px);
}

.about-us__page {
  padding: 10vh 0 10vh 0;
  color: #ffffff;
  &-photo {
    background: url('../../assets/images/Chicago_70230862_1050x590_tcm-33720.jpg') center center / cover;
    height: 30rem;
  }

  h1 {
    font-size: 3em;
  }
}