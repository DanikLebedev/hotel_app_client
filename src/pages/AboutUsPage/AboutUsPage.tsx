import React from 'react';
import './AboutUsPage.scss';
import { Container, Row, Col } from 'react-bootstrap';
import { config } from '../../config';
import galleryPhoto1 from '../../assets/images/7112.jpg';
import galleryPhoto2 from '../../assets/images/dui21-deluxe-room1.add1.low-res.jpg';
import galleryPhoto3 from '../../assets/images/3-high-res-restaurant-30.jpg';
import galleryPhoto4 from '../../assets/images/528941482_200fcf18b2_b.jpg';


export const AboutUsPage: React.FC = () => {
    return (
        <>
            <video className="videoTag" autoPlay loop muted>
                <source src={config.baseUrl + 'video.mp4'} type="video/mp4" />
            </video>
            <Container className="about-us__page">
                <h1 className="text-white text-center">About us</h1>
                <Row>
                    <Col lg={6} className="about-us__page-photo" md={6} sm={6}></Col>
                    <Col lg={6} md={6} sm={6} className="about-us__page-text">
                        <h3>We are the best</h3>
                        <p>
                            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque dolor ex ipsam mollitia nam
                            neque nobis numquam placeat tempore voluptates! Aut cum dolores inventore iusto nam natus
                            praesentium sit suscipit!
                        </p>
                    </Col>
                </Row>
            </Container>
            <Container fluid={true}>
                <Row className={'about-us__page-gallery'}>
                    <div className="grid">
                        <figure className="effect-layla">
                            <img src={galleryPhoto1} alt="img1" />
                            <figcaption>
                                <h2>
                                    Best <span>Prices</span>
                                </h2>
                                <p>We offered the best prices for best rooms</p>
                            </figcaption>
                        </figure>
                        <figure className="effect-layla">
                            <img src={galleryPhoto2} alt="img03" />
                            <figcaption>
                                <h2>
                                    Honest and HardWorking <span>Stuff</span>
                                </h2>
                                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi, tempora!</p>
                            </figcaption>
                        </figure>
                        <figure className="effect-layla">
                            <img src={galleryPhoto3} alt="img03" />
                            <figcaption>
                                <h2>
                                    Delicious <span>Food</span>
                                </h2>
                                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. In, vitae.</p>
                            </figcaption>
                        </figure>
                        <figure className="effect-layla">
                            <img src={galleryPhoto4} alt="img03" />
                            <figcaption>
                                <h2>
                                    Best <span>Choice</span>
                                </h2>
                                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Delectus, quas.</p>
                            </figcaption>
                        </figure>
                    </div>
                </Row>
            </Container>
        </>
    );
};
