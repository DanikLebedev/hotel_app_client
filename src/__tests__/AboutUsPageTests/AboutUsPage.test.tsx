import React from 'react';
import AboutUsPage from '../../pages/AboutUsPage/AboutUsPage';
import { shallow } from 'enzyme';

describe('about us page tests', () => {
    it('should renders without crashind', function() {
        shallow(<AboutUsPage />);
    });
});
