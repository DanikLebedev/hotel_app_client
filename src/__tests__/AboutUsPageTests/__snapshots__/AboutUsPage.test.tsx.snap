// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about us page tests should renders without crashing 1`] = `
<AboutUsPage
  i18n={Object {}}
  t={[Function]}
  tReady={false}
/>
`;
